### ISSUE TITLE
qmgr does not print resources_max.procct

### ISSUE THREAD

**Issue by yshl** [adaptivecomputing/torque#306]
I set resources_max.procct.
````
> qmgr -c "set queue batch resources_max.procct=4"
````
But qmgr does not print it.
````
> qmgr -c "print queue batch"
#
# Create queues and set their attributes.
#
#
# Create and define queue batch
#
create queue batch
set queue batch queue_type = Execution
set queue batch enabled = True
set queue batch started = True
````

**Comment by knielson** [comment]
My understanding is that even though the value does not show up in qmgr it
is still getting enforced. I am looking at the issue right now to see what
we can do to fix the problem.

Ken

On Sat, Apr 25, 2015 at 12:15 PM, yshl <notifications@github.com> wrote:

> I set resources_max.procct.
> > qmgr -c "set queue batch resources_max.procct=4"
> But qmgr does not print it.
> > qmgr -c "print queue batch"
> #
> # Create queues and set their attributes.
> #
> #
> # Create and define queue batch
> #
> create queue batch
> set queue batch queue_type = Execution
> set queue batch enabled = True
> set queue batch started = True
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adaptivecomputing/torque/issues/306>.
>



-- 
       [image: Adaptive Computing] <http://www.adaptivecomputing.com>
 [image: Twitter] <http://twitter.com/AdaptiveMoab> [image: LinkedIn]
<http://www.linkedin.com/company/448673?goback=.fcs_GLHD_adaptive+computing_false_*2_*2_*2_*2_*2_*2_*2_*2_*2_*2_*2_*2&trk=ncsrch_hits>
[image:
YouTube] <http://www.youtube.com/adaptivecomputing> [image: GooglePlus]
<https://plus.google.com/u/0/102155039310685515037/posts> [image: Facebook]
<http://www.facebook.com/pages/Adaptive-Computing/314449798572695?fref=ts>
[image:
RSS] <http://www.adaptivecomputing.com/feed>
Ken Nielson Sr. Software Engineer
 +1 801.717.3700 office    +1 801.717.3738 fax
 1712 S. East Bay Blvd, Suite 300     Provo, UT 84606
 www.adaptivecomputing.com

**Comment by knielson** [comment]
A fix for this has been checked into 5.1-dev, 5.0-dev and develop.

606278b..ccdccd2 5.1-dev -> 5.1-dev
860161c..7b4dfcd 5.0-dev -> 5.0-dev
fb02124..5712ccb develop -> develop

**Comment by yshl** [comment]
Thank you.

**Comment by knielson** [comment]
This has been fixed in 5.1.1. It will be fixed in 5.0.3 as well.

Ken

On Sat, Jun 27, 2015 at 12:27 AM, yshl <notifications@github.com> wrote:

> Thank you.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adaptivecomputing/torque/issues/306#issuecomment-115974813>
> .
>



-- 
       [image: Adaptive Computing] <http://www.adaptivecomputing.com>
 [image: Twitter] <http://twitter.com/AdaptiveMoab> [image: LinkedIn]
<http://www.linkedin.com/company/448673?goback=.fcs_GLHD_adaptive+computing_false_*2_*2_*2_*2_*2_*2_*2_*2_*2_*2_*2_*2&trk=ncsrch_hits>
[image:
YouTube] <http://www.youtube.com/adaptivecomputing> [image: GooglePlus]
<https://plus.google.com/u/0/102155039310685515037/posts> [image: Facebook]
<http://www.facebook.com/pages/Adaptive-Computing/314449798572695?fref=ts>
[image:
RSS] <http://www.adaptivecomputing.com/feed>
Ken Nielson Sr. Software Engineer
 +1 801.717.3700 office    +1 801.717.3738 fax
 1712 S. East Bay Blvd, Suite 300     Provo, UT 84606
 www.adaptivecomputing.com
